var/spack/repos/builtin/packages/r-amap/package.py

```diff
@@ -12,7 +12,7 @@ class RAmap(RPackage):
     Tools for Clustering and Principal Component Analysis (With robust methods,
     and parallelized functions)."""
 
-    homepage = "http://mulcyber.toulouse.inra.fr/projects/amap/"
+    homepage = "https://cran.r-project.org/web/packages/amap/index.html"
     url      = "https://cloud.r-project.org/src/contrib/amap_0.8-16.tar.gz"
     list_url = "https://cloud.r-project.org/src/contrib/Archive/amap/"
```

Comment:
For R packages we really need a `cran` attribute similar to `pypi` that autopopulates all of these attributes. @glennpj are you interested in working on this?

Reply:
Sure, I actually planned on doing that when I do my next wave of CRAN package updates. I have not been pushing for that in PRs though.

Reply:
Just to make sure that I am clear about the goal, are we looking to have the `cran` attribute in all CRAN packages? The attribute already exists but its usage has not been standardized on as of yet.

Reply:
Maybe y'all want to open an issue on this instead of the thread getting lost in this PR? There are already so many comments I can't find ones that I'm looking for with that accordian thing GitHub is doing!